src/types/exports.ts

```diff
@@ -96,7 +96,9 @@ export interface EdgeFakeWorldProps extends CommonProps {
  * We don't want this library to depend on `@types/react`,
  * since that isn't relevant for our Node or browser builds.
  */
-type ComponentType<Props> = React$ComponentType<Props>
+type ComponentType<Props> = (
+  props: Props
+) => { type: any; props: any; key: string | number | null }
```

Comment:
Should we not use `React.ComponentType`; if not, why?

Reply:
We don't want to make `@types/react` a dependency, since that isn't relevant for the Web or Node targets.